latest/files.json

```diff
@@ -1 +1 @@
d.png","Samsung Galaxy S9 Coral Blue.png","Google Pixel 3 Not Pink.png","Google Pixel 3 Just Black.png","Google Pixel 3 Clearly White.png","Nexus 6P.png","Samsung Galaxy S8 Arctic Silver.png","Samsung Galaxy S8 Orchid Gray.png","Samsung Galaxy S8 Midnight Black.png","Samsung Galaxy S8 Coral Blue.png","Samsung Galaxy S8 Maple Gold.png","Nexus 5x.png","Samsung Galaxy Note 5 Black.png","Samsung Galaxy Note 5 Pink.png","Samsung Galaxy Note 5 White.png","Samsung Galaxy Note 5 Silver Titanium.png","Samsung Galaxy Note 5 Gold.png","Apple iPhone 6s Plus Space Gray.png","Apple iPhone 6s Plus Rose Gold.png","Apple iPhone 6s Plus Silver.png","Apple iPhone 6s Plus Gold.png","Apple iPhone 7 Rose Gold.png","Apple iPhone 7 Gold.png","Apple iPhone 7 Silver.png","Apple iPhone 7 Jet Black.png","Apple iPhone 7 Matte Black.png","Apple iPhone 8 Gold.png","Apple iPhone 8 Silver.png","Apple iPhone 8 Space Gray.png","Samsung Galaxy S Duos.png","Apple iPhone 5c Green.png","Apple iPhone 5c Blue.png","Apple iPhone 5c Red.png","Apple iPhone 5c White.png","Apple iPhone 5c Yellow.png","Apple iPhone 8 Plus Gold.png","Apple iPhone 8 Plus Space Gray.png","Apple iPhone 8 Plus Silver.png","HTC One A9 White.png","HTC One A9 Black.png","Google Pixel 3 XL Clearly White.png","Google Pixel 3 XL Just Black.png","Google Pixel 3 XL Not Pink.png","Nexus 4.png","Motorola Moto E Black.png","Motorola Moto E White.png","Huawei P8 White.png","Huawei P8 Black.png","Samsung Galaxy S3.png","Samsung Galaxy S5 White.png","Samsung Galaxy S5 Black.png","Apple iPhone XS Space Gray.png","Apple iPhone XS Gold.png","Apple iPhone XS Silver.png","Apple iPhone X Silver.png","Apple iPhone X Space Gray.png","Samsung Galaxy Grand Prime White.png","Samsung Galaxy Grand Prime Gold.png","Samsung Galaxy Grand Prime Black.png","Apple iPhone XS Max Space Gray.png","Apple iPhone XS Max Gold.png","Apple iPhone XS Max Silver.png","Apple iPhone 7 Plus Rose Gold.png","Apple iPhone 7 Plus Gold.png","Apple iPhone 7 Plus Matte Black.png","Apple iPhone 7 Plus Silver.png","Apple iPhone 7 Plus Jet Black.png","Apple iPhone XR Blue.png","Apple iPhone XR Space Gray.png","Apple iPhone XR Silver.png","Apple iPhone XR Yellow.png","Apple iPhone XR Red.png","Apple iPhone XR Coral.png","Apple iPhone 5s Gold.png","Apple iPhone 5s Space Gray.png","Apple iPhone 5s Silver.png","Apple iPhone SE Gold.png","Apple iPhone SE Rose Gold.png","Apple iPhone SE Silver.png","Apple iPhone SE Space Gray.png","Apple iPhone 6s Rose Gold.png","Apple iPhone 6s Gold.png","Apple iPhone 6s Silver.png","Apple iPhone 6s Space Gray.png","HTC One M8 Silver.png","HTC One M8 Black.png","HTC One M8 Gold.png","Motorola Moto G.png","Apple iPad Pro Gold.png","Apple iPad Pro Silver.png","Apple iPad Pro Space Gray.png","Apple iPad Air 2 Space Gray.png","Apple iPad Air 2 Gold.png","Apple iPad Air 2 Silver.png","Nexus 9.png","Apple iPad Mini 4 Gold.png","Apple iPad Mini 4 Silver.png","Apple iPad Mini 4 Space Gray.png","Apple-Macbook-Gold.png","Apple-Macbook-Space-Gray.png"]
```

Comment:
What defines the ordering here in this files (also offset.json)? Seems Android and iOS are intermixed right now.

Reply:
I think this order is caused by the three Dir commands in frames_generator whose ordering is unspecified. [But according to this SO](https://stackoverflow.com/a/7390363/2380518) it should be relatively easy to sort them.

Reply:
Sorting by name would probably be best here, wdyt?

Reply:
I think it would make the diffs clearer in the future. So I will modify the frames_generator and make one more commit here with new sorted JSON files and one in fastlane with updated frames_generator.